README.md

```diff
@@ -1 +1,176 @@
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML & CSS
+- JavaScript
+- React & Redux.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live Demo Link](https://world-countries.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Download Visual Studio Code.
+- Download Git.
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+1. Clone the repo
+```sh
+   git clone https://github.com/odaialazzeh/world-countries.git
+```
+
+
+### Install
+
+Install this project with:
+
+1. Install NPM packages
+```sh
+   npm install
+```
+
+### Run tests
+
+```sh
+
+ Type the following command in your terminal: npm start 
+
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Odai**
+
+- GitHub: [odai-alazzeh](https://github.com/odaialazzeh)
+- Twitter: [@twitterhandle](https://twitter.com/odaialazzeh4)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/odaialazzeh/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add search country by voice
+- Allow user to add new data to API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/odaialazzeh/math-magicians/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+A list of resources that i find helpful and would like to give credit to :
+
+- [Microverse ](https://www.microverse.org)
+- [GitHub ](https://www.github.com)
+- [Youtube ](https://www.youtube.com)
+- [W3schools ](https://www.w3schools.com)
+- [Stackoverflow ](https://stackoverflow.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+
+- **How to open the World Countries details?**
+
+  - *You can click/tap on the country name or card it will navigate to details report page.*
+
+- **How to return to the home page?**
+
+  - *You can click the left arrow located in the top-left corner of the page.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License) licensed.
+
```

Comment:
- [OPTIONAL] The link to your _**MIT**_ redirects to an error page. The name of your license file is `LICENSE` and not **_License_**. Kindly update the name. 👍